### DISCUSSION TITLE
使用一键部署客户端之后无法运行

### DISCUSSION THREAD

**Discussion by Cyrker** [zdz/ServerStatus-Rust#76] [Q&A] [answered]
系统是 debian10
curl -sSf "http://urlxxx/report/i?pass=p&uid=uid&vnstat=1" | bash 
运行这个之后能正常安装客户端  但是无法正常运行
报错为
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', client/src/status.rs:116:43
stack backtrace:
   0:     0x7fcecae1c000 - <unknown>
   1:     0x7fcecacc99dc - <unknown>
   2:     0x7fcecadf25c0 - <unknown>
   3:     0x7fcecae1cd05 - <unknown>
   4:     0x7fcecae1db73 - <unknown>
   5:     0x7fcecae1d6ab - <unknown>
   6:     0x7fcecae1d646 - <unknown>
   7:     0x7fcecae1d602 - <unknown>
   8:     0x7fcecac55292 - <unknown>
   9:     0x7fcecac5538c - <unknown>
  10:     0x7fcecac78924 - <unknown>
  11:     0x7fcecac79016 - <unknown>
  12:     0x7fcecac838c9 - <unknown>
  13:     0x7fcecac75ac6 - <unknown>
  14:     0x7fcecac79f2a - <unknown>
  15:     0x7fcecac5e2cc - <unknown>
  16:     0x7fcecac85018 - <unknown>
Aborted
刚开始我以为是下载的版本和系统不兼容..后来检查了下发现是 vnstat=1 这个参数的问题
只要带这个参数就会闪退..不太清楚原理 求解

**Comment by zdz** [comment, marked as answer]
https://doc.ssr.rs/vnstat/  先确认一下 vnstat 版本

**Reply by Cyrker** [reply]
原来如此.....
apt安装的vnstat是1.18版本的..
